docs/.eslintrc.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  root: true,
```

Comment:
Ensures ESLint stops walking upwards so we know which config we're getting.

(We'll soon be using flat config but we'll cross that bridge when we get there.)